src/main/java/core/basesyntax/Hello_World.java

```diff
@@ -1,43 +1,43 @@
 package core.basesyntax;
 
-import core.basesyntax.Model.Dog;
-import core.basesyntax.Model.Cat;
-
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
 import java.time.LocalDate;
-import java.util.Objects;
-
-public class Hello_World
-{
-    private String a;
-    private int vARIaBLE;
-    private int age, size;
-
 
+public class HelloWorld {
```

Comment:
your code does not compile because you didn't rename the file where this class is located